### DISCUSSION TITLE
should not have cache for `useRefetch`?

### DISCUSSION THREAD

**Discussion by himself65** [wakujs/waku#503] [General]

**Comment by dai-shi** [comment]
for static files?

**Comment by himself65** [comment]
for... Idk, actually there is no document for this hook, and I don't know that's the correct input. let me read the source code first

**Comment by himself65** [comment]
I see. `useRefetch` will rerender the whole RSC for each call. So I have other questions like:

- Since react caches the fetch, how could I get the cached value after each RSC render so that I can put it somewhere (for example, redis) and put it back before the next UI render?
